pygmt/src/grdhisteq.py

```diff
@@ -281,12 +285,12 @@ def compute_bins(
 
         Returns
         -------
-        ret: pandas.DataFrame or None
-            Return type depends on the ``outfile`` parameter:
-
-            - pandas.DataFrame if ``outfile`` is True or None
-            - None if ``outfile`` is a str (file output is stored in
+        ret : pandas.DataFrame or numpy.ndarray or None
+            Return type depends on ``outfile`` and ``output_type``:
+            - None if ``outfile`` is set (output will be stored in file set by
```

Comment:
Extra line needed to render properly
```suggestion

            - None if ``outfile`` is set (output will be stored in file set by
```